src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -48,6 +52,17 @@ public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAl
             _keepAliveDuration = (int)keepAliveInterval.TotalMilliseconds * (Stopwatch.Frequency / 1000);
         }
 
+        internal StreamTracker _streamTracker
```

Comment:
nit: call this `StreamTracker` and the field `_streamTracker`

Reply:
I felt uncomfortable overriding the class name, but it is nicer this way